src/components/sbb-slider/sbb-slider.scss

```diff
@@ -0,0 +1,141 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-slider-line-height: #{px-to-rem-build(4)};
+  --sbb-slider-value-fraction: 0;
+  --sbb-slider-knob-size: #{px-to-rem-build(28)};
+  --sbb-slider-knob-border-size: #{px-to-rem-build(3)};
+  --sbb-slider-knob-color: var(--sbb-color-white-default);
+  --sbb-slider-knob-border-color: var(--sbb-color-red-default);
+  --sbb-slider-icon-color: var(--sbb-color-black-default);
+  --sbb-slider-selected-line-color: var(--sbb-color-red-default);
+  --sbb-slider-selected-line-disabled-color: var(--sbb-color-smoke-default);
+  --sbb-slider-line-color: var(--sbb-color-smoke-default);
+  --sbb-slider-line-disabled-color: var(--sbb-color-graphite-default);
+
+  @include ifForcedColors {
+    --sbb-slider-selected-line-color: Highlight;
+    --sbb-slider-line-color: CanvasText;
+  }
+
+  display: inline;
+}
+
+:host([data-sbb-form-field]) {
+  --form-field-overflow: visible;
+  --form-field-background-color: transparent;
+}
+
+// override icons color
+:host([disabled]) {
+  --sbb-slider-icon-color: var(--sbb-color-graphite-default);
+}
+
+:host([readonly]) {
+  --sbb-slider-icon-color: var(--sbb-color-smoke-default);
+}
+
+:host([disabled]),
+:host([readonly]) {
+  @include ifForcedColors {
+    --sbb-slider-icon-color: GrayText;
+    --sbb-slider-selected-line-disabled-color: GrayText;
+    --sbb-slider-line-disabled-color: GrayText;
+  }
+}
+
+.sbb-slider__wrapper {
+  max-width: #{px-to-rem-build(400)};
+  min-width: #{px-to-rem-build(150)};
+  width: 100%;
+  display: flex;
+  gap: var(--sbb-spacing-fixed-2x);
+  padding-block: var(--sbb-spacing-fixed-1x);
+  color: var(--sbb-slider-icon-color);
+}
+
+.sbb-slider__container {
+  position: relative;
+  min-height: var(--sbb-size-icon-ui-small);
+  flex-grow: 1;
+}
+
+.sbb-slider__range-input {
+  position: absolute;
+  width: 100%;
+  inset: 0;
+  opacity: 0;
+  margin: 0;
+}
+
+.sbb-slider__line {
+  @include absoluteCenterY;
+
+  height: var(--sbb-slider-line-height);
+  width: 100%;
+  background-color: var(--sbb-slider-line-color);
+  border-radius: var(--sbb-slider-line-height);
+  overflow: hidden;
+  pointer-events: none;
+
+  :host([disabled]) &,
+  :host([readonly]) & {
+    --sbb-slider-line-color: var(--sbb-slider-line-disabled-color);
+  }
```

Comment:
I'd prefer this definition directly on host level, I mean at the moment at line 42